graphics/diagrams/seamless-posting/img-2.svg

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 19.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="图层_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"

```

Comment:
I'm guessing your software put Chinese characters in for the ID?

Side note: I personally feel lucky that my native language dominates software, but I find it impressive when people learn English _and_ coding, open source software, etc. Good job!